Sources/KlaviyoLocation/KlaviyoLocationManager.swift

```diff
@@ -8,21 +8,29 @@
 import Combine
 import CoreLocation
 import Foundation
+import KlaviyoCore
 import KlaviyoSwift
 import OSLog
 
 public class KlaviyoLocationManager: NSObject {
     static let shared = KlaviyoLocationManager()
 
-    private let locationManager = CLLocationManager()
+    private var locationManager: LocationManagerProtocol
```

Comment:
```suggestion
    private let locationManager: LocationManagerProtocol
```
can this be a constant?